src/material/datepicker/calendar-body.ts

```diff
@@ -309,8 +312,8 @@ export class MatCalendarBody<D = any> implements OnChanges, OnDestroy, AfterView
    * Adding delay also complicates writing tests.
    */
   _focusActiveCell(movePreview = true) {
-    this._ngZone.runOutsideAngular(() => {
-      this._ngZone.onStable.pipe(take(1)).subscribe(() => {
+    afterNextRender(
+      () => {
         setTimeout(() => {
```

Comment:
Could we get rid of the `setTimeout` here?

Reply:
I originally tried it without the setTimeout and the TGP had failures, so I put it back